model/src/w3src3md.F90

```diff
@@ -1255,7 +1255,14 @@ SUBROUTINE W3SDS3 (A, K, CG, EMEAN, FMEAN, WNMEAN, USTAR, USDIR,    &
         DOUT(IK,ITH) = D(ITH+(IK-1)*NTH)
       END DO
     END DO
+<<<<<<< HEAD
```

Comment:
Hi @MathieuDutSik, could you resolve this conflict then I can start the review?

Reply:
Sorry for that mistake

Reply:
It has been corrected.

Reply:
No worries, thanks @MathieuDutSik!  I'll start reviewing.